.github/workflows/publish-next.yml

```
@@ -145,8 +145,9 @@ jobs:
         run: yarn build
         working-directory: ./canary
 
-  log-failure-metric:
-    # Send data point 1 to metric PublishNextFailure, if  it's a failure
+  log-failure-metric-prod:
+    # to be deprecated once log-failure-metric is up
+    # Send data point 1 to metric PublishNextFailure in prod us-east-1, if  it's a failure
```

I'm sorry, I keep finding a new one 😓 
```suggestion
    # Send data point 1 to metric PublishNextFailure in prod us-east-1, if it's a failure
```
There's one more in line 186

---

Thanks. My fault 😭